### ISSUE TITLE
cubbing runes

### ISSUE THREAD

**Issue by leetzor** [kolton/d2bot-with-kolbot#284]
Hello. I want that bot "cubing" runes for crafting, for exaple nef rune. But he cube 3 tir, and than dropped it on ground. Maybe i need to cancel cubing section for runes in char. config ?

**Comment by LeMule** [comment]
Do you have him set to keep the runes? Probably why he's dropping it.

**Comment by leetzor** [comment]
Nope, i just made it for cubing right runes to make rerolls faster. Didnt make keep it in kolton pickit.

**Comment by leetzor** [comment]
Or bot cubing runes for craft itself ? For example if need nef, it has been cubing 3 tir or only pick nef or other runes for craft ?

**Comment by leetzor** [comment]
Thanks LeMule. I understood, and made pickit for crafting in other way, thanks :)
